distributed/spill.py

```diff
@@ -161,21 +153,17 @@ def handle_errors(self, key: str | None) -> Iterator[None]:
             (key_e,) = e.args
             assert key_e in self.fast
             assert key_e not in self.slow
-            now = perf_counter()
```

Comment:
There's no guarantee for perf_counter to be monotonic or UTC according to the Python API spec. So this could have potentially muted the log for 1 hour when switching timezone due to daylight savings.